lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveRecord::Acts::List::SequentialUpdatesDefiner #:nodoc:
+  def self.call(caller_class, column, sequential_updates_option)
+    caller_class.class_eval do
+      @sequential_updates = nil
+
+      define_method :sequential_updates? do
```

Comment:
> So you can just do

Nope, calling sequential_updates? that way will produce 'undefined local variable or method `sequential_updates_option'. Cause, well, sequential_updates_option would be really not a local variable nor method (undefined). This definer trick makes it available for the sequential_updates? **block**, cause block do have access to the caller scope variables.

Another possible option is to store the whole acts_as_list configuration as a class variable after initialization. But this is not in the style of this gem. 

Please check [a perfect example](https://github.com/swanandp/acts_as_list/blob/master/lib/acts_as_list/active_record/acts/add_new_at_method_definer.rb) of doing the same thing inside another definer.